lib/scryptenc/scryptenc.h

```diff
@@ -62,23 +62,22 @@
  * compared to the computed limits and an error is returned if decrypting
  * the data would take too much memory or CPU time.
  */
-/**
- * Return codes from scrypt(enc|dec)_(buf|file):
- * 0	success
- * 1	getrlimit or sysctl(hw.usermem) failed
- * 2	clock_getres or clock_gettime failed
- * 3	error computing derived key
- * 4	could not read salt from /dev/urandom
- * 5	error in OpenSSL
- * 6	malloc failed
- * 7	data is not a valid scrypt-encrypted block
- * 8	unrecognized scrypt format
- * 9	decrypting file would take too much memory
- * 10	decrypting file would take too long
- * 11	password is incorrect
- * 12	error writing output file
- * 13	error reading input file
- */
+
+/* Return codes from scrypt(enc|dec)_(buf|file|prep): */
```

Comment:
Is the indentation weird here, or is it just github being stupid?

Reply:
I think the indentation looks ok in both the `.h` and in github...?  The previous content had an extra space (because it was a multiline comment), whereas this is a one-line comment.

Some of the later lines do wrap around, such as
```
#define SCRYPT_ELIMIT           1       /* getrlimit or sysctrl(hw.usernamem faile
d */
```
which is distinctly sub-optimal.  Now that you're ok with this in theory, I'll play around with a few ways to attempt to avoid that.